tracer/src/Datadog.Trace/Ci/CIVisibility.cs

```diff
@@ -84,10 +84,12 @@ public static void Initialize()
             var eventPlatformProxyEnabled = false;
             if (!settings.Agentless)
             {
-                if (settings.ForceAgentsEvpProxy)
+                if (!string.IsNullOrWhiteSpace(settings.ForceAgentsEvpProxy))
                 {
                     // if we force the evp proxy (internal switch)
-                    EventPlatformProxySupport = EventPlatformProxySupport.V2;
+                    EventPlatformProxySupport = Enum.TryParse<EventPlatformProxySupport>(settings.ForceAgentsEvpProxy, out var parsedValue) ?
```

Comment:
Enum.TryParse is slow right - if you add the `[EnumExtensions]` attribute to `EventPlatformProxySupport` you can use the fast version 😄

Reply:
Oh right!, will do that...

Reply:
the source generator doesn't contain `ParseFast` or similar, so I'll leave it like this.